webui/src/pages/repositories/repository/objectViewer.tsx

```diff
@@ -1,147 +1,166 @@
-    fileExtension: string;
-    sizeBytes: number;
-    showFullNavigator?: boolean;
-    presign?: boolean;
+  repoId: string;
+  refId: string;
+  path: string;
+  loading: boolean;
+  error: Error | null;
+  contentType?: string | null;
+  fileExtension: string;
+  sizeBytes: number;
+  showFullNavigator?: boolean;
+  presign?: boolean;
 }
 
-
 export const Loading: FC = () => {
-    return (
-        <Alert variant={"info"}>Loading...</Alert>
-    );
+  return <Alert variant={"info"}>Loading...</Alert>;
 };
 
 export const getFileExtension = (objectName: string): string => {
-    const objectNameParts = objectName.split(".");
-    return objectNameParts[objectNameParts.length - 1];
-}
+  const objectNameParts = objectName.split(".");
+  return objectNameParts[objectNameParts.length - 1];
+};
 
 export const getContentType = (headers: Headers): string | null => {
-    if (!headers) return null;
+  if (!headers) return null;
 
-    return headers.get("Content-Type") ?? null;
-}
+  return headers.get("Content-Type") ?? null;
+};
 
 const FileObjectsViewerPage = () => {
-    const { repoId } = useParams<ObjectViewerPathParams>();
-    const queryString = useQuery<ObjectViewerQueryString>();
-    const refId = queryString["ref"] ?? "";
-    const path = queryString["path"] ?? "";
-    const {response, error, loading} = useAPI( () => {
-            return objects.head(repoId, refId, path)
-        },
-        [repoId, refId, path]);
-
-    let content;
-    if (loading) {
-        content  = <Loading/>;
-    } else if (error) {
-        content = <Error error={error}/>
-    } else {
-        const fileExtension = getFileExtension(path);
-        // We'll need to convert the API service to get rid of this any
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const contentType = getContentType((response as any)?.headers);
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const sizeBytes = parseInt((response as any)?.headers.get('Content-Length'))
-        content = <FileContents
-            repoId={repoId || ''}
+  const { repoId } = useParams<ObjectViewerPathParams>();
+  const queryString = useQuery<ObjectViewerQueryString>();
+  const refId = queryString["ref"] ?? "";
+  const path = queryString["path"] ?? "";
+  const { response, error, loading } = useAPI(() => {
+    return objects.head(repoId, refId, path);
+  }, [repoId, refId, path]);
+
+  let content;
+  if (loading) {
+    content = <Loading />;
+  } else if (error) {
+    content = <Error error={error} />;
+  } else {
+    const fileExtension = getFileExtension(path);
+    // We'll need to convert the API service to get rid of this any
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const contentType = getContentType((response as any)?.headers);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const sizeBytes = parseInt(
+      (response as any)?.headers.get("Content-Length")
```

Comment:
The linter - "Unexpected any. Specify a different type"

Reply:
Ooops, reformatting moved the line the `eslint` comment was for one line down
Fixed.